policyengine_us/variables/gov/states/ca/tax/income/alternative_minimum_tax/amti/ca_amti_total_adjustments.py

```diff
@@ -0,0 +1,28 @@
+from policyengine_us.model_api import *
+
+
+class ca_amti_total_adjustments(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = (
+        "California total adjustments and preferences for the AMTI calculation"
+    )
+    defined_for = StateCode.CA
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.ftb.ca.gov/forms/2022/2022-540-p.pdf"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.ca.tax.income.alternative_minimum_tax
+
+        # line 14
+        return where(
+            tax_unit("ca_itemized_deductions", period) > 0,
+            add(tax_unit, period, p.amti_sources)
+            - tax_unit(
+                "ca_standard_deduction", period
+            ),  # if itemized, line 1 = 0
+            add(
+                tax_unit, period, p.amti_sources
```

Comment:
not sure about this but maybe it means that if not itemized, only take `ca_standard_deduction` and skip line 2-5?
![image](https://github.com/PolicyEngine/policyengine-us/assets/31402450/c60fcb6b-3f0b-4820-984f-f6a22d7b56b1)